images/runtime/php/8.1/base.bullseye.Dockerfile

```diff
@@ -1,11 +1,9 @@
-# DisableDockerDetector "Below image not yet supported in the Docker Hub mirror"
-FROM php:8.1-bullseye
+FROM oryxdevmcr.azurecr.io/private/oryx/php-8.1
```

Comment:
Is this the image we want to use here? It was previously being pulled from Docker Hub, but hadn't been migrated to the [approved MCR mirror](https://eng.ms/docs/more/containers-secure-supply-chain/approved-images) yet